runtime/hydradx/src/weights/pallet_dispatcher.rs

```diff
@@ -90,4 +90,9 @@ impl<T: frame_system::Config> pallet_dispatcher::WeightInfo for HydraWeight<T> {
         Weight::from_parts(3_864_000, 0)
             .saturating_add(T::DbWeight::get().writes(1_u64))
     }
+
+    fn dispatch_with_extra_gas(n: u32) -> Weight {
+        Weight::from_parts(3_864_000, 0)
+            .saturating_add(T::DbWeight::get().writes(1_u64))
```

Comment:
is there really a write?

Reply:
no, i dont thin so - i havent yet benchmarked, so this is just a random copy pasta from another extrinaic. will update

Reply:
updated weights - there seems to be a 1 write. but looking at the dispatch functions - they all do one 1 write.